docs/metastore_upgrade_guide.md

```diff
@@ -0,0 +1,210 @@
+| DBFS_ROOT_NON_DELTA | Tables saved to the DBFS file system that are not in Delta format                                                                                                                  | The current upgrade process will create a managed table using CTAS                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| VIEW                | Datbase Views                                                                                                                                                                      | Views are recreated during the upgrade process. The view's definition will be modified to repoint to the new UC tables. Views should be migrated only after all the dependent tables have been migrated. The upgrade process account for View to View dependencies.                                                                                                                                                                                                                                                                                                                   |
+
+The upgrade process can be triggered using the following command:
+```bash
+databricks labs ucx migrate-tables
+```
+
+Or by running the "Migrate Tables" workflow deployed to the workspace.
```

Comment:
```suggestion
Or by running the "Migrate Tables" workflow deployed to the workspace.
```
we have multiple workflows deployed as part of ucx

Reply:
Fixed